packages/core/README.md

```diff
@@ -220,16 +230,66 @@ A full documentation of the masterportalapiPolygonFillHatch is available at the
 >|patternColor|no|Number[]|`[255, 255, 255, 1]`|Fill color of pattern drawn on polygon.|
 >|size|no|Number|`30`|Edge length of a singular repeated pattern element.|
 
-##### mapConfiguration.LayerConf
+##### <...masterportalapi.fields>
+
+The `<...masterportalapi.fields>` means that any masterportalapi field may also be used here _directly_ in the mapConfiguration. The fields described here are fields that are interesting for the usage of POLAR.
+Fields that are not set as required have default values.
+
+| fieldName | type | description |
+| - | - | - |
+| layerConf | layerConf | Layer configuration of all available layers as a service register. Layers defined here are not directly shown in a client, see `mapconfiguration.layers` for that. |
+| layers | layer[] | Configuration of layers that are supposed to be used in the respective client. All layers defined here have to have an entry in `mapConfiguration.layerConf`. If `@polar/plugin-layer-chooser` is installed and configured, all these layers will be displayed in that menu. |
+| startCenter | number[] | Initial center coordinate. Needs to be defined in the chosen leading coordinate system. |
+| backgroundImage | string? | Image to be displayed as a background of the map. Defaults to `''`, so no image being used. |
```

Comment:
This effectively is supposed to change how some clients are rendered initially. On slow devices or when the WMS is slow, this does make a difference. I'm okay with `''` being the default, but now clients need the image configured.

Theoretically.

Practically, the backgroundImage feature doesn't work anyway, see e.g. the implementation of `setBackgroundImage` in the Meldemichel, where `"masterportalAPI's setBackgroundImage doesn't work in nested shadow roots."` is noted.

Reply:
Interestingly enough, Masterportal has switched to defaulting to no backgroundImage, thus this is just the default of `@masterportal/masterportalapi`.
I've just added this here for completion.

I'm not quite sure what I am supposed to change here. Could you elaborate?

Reply:
I'm not sure how to handle this, but now effectively something's documented that doesn't work. I'd either skip documenting it at all, or propose it's fixed (or it's noted somewhere that it needs fixing). However, I remember 0 instances of anyone needing this backgroundImage, so I guess it's safe to drop the doc.

Reply:
Removed it be841eaf642863a410e8a3747cbcddefbb09abe5